php-nginx/README.md

```diff
@@ -96,6 +96,15 @@ WEB_HTTPS_ONLY      | Whether to redirect all HTTP traffic to HTTPS | true/false
 WEB_REVERSE_PROXIED | Whether to interpret X-Forwarded-Proto as the $custom_scheme and $custom_https emulation. | true/false | true
 WEB_SSL_FULLCHAIN | The location of the SSL certificate and intermediate chain file | absolute filename | /etc/ssl/certs/fullchain.pem
 WEB_SSL_PRIVKEY | The location of the SSL private key file | absolute filename | /etc/ssl/private/privkey.pem
+ASSETS_ENV | The assets environment assets are downloaded/applied from. If unset no asset actions will happen | a asset environment | 
```

Comment:
Can't we just have `ASSETS_PATH `? To me that looks like oversimplifying a bit and hiding a very simple thing which is `tools/assets/X`. So it'll be clear for everybody that we are un-tarring the archives at this location :)